rskj-core/src/test/java/co/rsk/signing/KeyFileHandlerTest.java

```diff
@@ -0,0 +1,45 @@
+package co.rsk.signing;
+
+import org.apache.commons.lang3.StringUtils;
+import org.ethereum.crypto.ECKey;
+import org.junit.Assert;
+import org.junit.Test;
+import org.spongycastle.util.encoders.Hex;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * Created by mario on 30/08/2016.
+ */
+public class KeyFileHandlerTest {
+    private static final String TEST_KEY_FILE = "src/test/resources/peg/federate.test.key";
```

Comment:
We're also doing IO here, is there any alternative?

Reply:
Ditto previous comment.

Reply:
Ditto previous comment :-)